trpcultivate_phenotypes/src/TripalCultivateValidator/ValidatorTraits/Project.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Drupal\trpcultivate_phenotypes\TripalCultivateValidator\ValidatorTraits;
+
+use Drupal\tripal_chado\Controller\ChadoProjectAutocompleteController;
+
+/**
+ * Provides setters focused for setting a project used by the importer
+ * to package datasets, and getter to retrieve the set value.
+ */ 
+trait Project {
+  /**
+   * The key used by the setter method to create a project element 
+   * in the context array, as well as the key used by the getter method 
+   * to reference and retrieve the project element value.
+   * 
+   * @var string
+   */
+   private string $trait_key = 'project';
+
+  /**
+   * Sets a project.
```

Comment:
What about this wording?


```suggestion
   * Sets a single project for use by a validator.
   *
   * Note: This project must exist in the chado.project table already and as long as it
   * does both the project_id and project name will be saved for use by the validator.
```